lib/grpc/server/adapter.ex

```diff
@@ -0,0 +1,16 @@
+defmodule GRPC.Server.Adapter do
+  @moduledoc """
+  HTTP server adapter for GRPC.
+  """
+
+  alias GRPC.Server.Adapters.Cowboy.Handler
+
+  @callback start(atom, %{String.t() => [module]}, non_neg_integer(), Keyword.t()) ::
+              {atom, any, non_neg_integer}
+
+  @callback stop(atom, %{String.t() => [module]}) :: :ok | {:error, :not_found}
```

Comment:
```suggestion
  @callback stop(atom(), %{String.t() => [module()]}) :: :ok | {:error, :not_found}
```

Reply:
I think we can standardize on having parens :)

Reply:
Agreed :)

Reply:
Agree and done (at least everywhere I see 😂 ), but I think that we should think about how we can add a validation about that 😅